### DISCUSSION TITLE
Questions regarding DIA-NN smart-profiling

### DISCUSSION THREAD

**Discussion by hguturu** [vdemichev/DiaNN#317] [Q&A] [answered]
DIA-NN documentation recommends using `--smart-profiling` and the documentation says it “enables an intelligent algorithm which determines how to extract spectra, when creating a spectral library from DIA data. This is highly recommended and should almost always be enabled.” 

Questions:
- Does this flag do anything when a library is not being created?
- Is a library always created from DIA data for every run?
- If a library is created, is the library from one sample used on another? i.e. would our peptide sensitivity increase with larger group runs?

**Comment by vdemichev** [comment, marked as answer]
- No.
- No, when DIA-NN is instructed to create a library, the library is created from the entire experiment.
- Yes, typically having more runs in the experiment will result in a better library, provided the runs are heterogeneous and not replicate injections.

**Reply by hguturu** [reply]
Got it. Thanks for clarifying. As I mentioned in the previous thread, we currently run all our DIA-NN runs with `--smart-profiling`. I guess we should remove that to not complicate things.

e.g.

> /usr/diann/1.8/diann-1.8 --threads 8 --dir raw --fasta UP000005640_9606_combined.fasta --lib UP000005640_9606_combined.predicted.speclib --use-quant --mass-acc-ms1 10 --mass-acc 10 --qvalue 0.01 --matrices --missed-cleavages 1 --met-excision --cut K*,R* ~~-smart-profiling~~ --relaxed-prot-inf --pg-level 1 --reannotate
